### PR TITLE
Update Rubocop definitions and appease the cops

### PR DESCRIPTION
**Pull request by mjgiarlo**

### CODE REVIEW

File: lib/dor/rights_auth.rb
Comment (jmartin-sul) [inline lib/dor/rights_auth.rb:428]: i can't tell whether the branches that were consolidated in this PR's change were intended to be kept separate.  that is, if `index_terms` includes `cdl_none` as well as `no_read_machine` or `none_read`, should the `cdl_none` take precedence, leading this method to produce a result of `'controlled digital lending'` instead of `'dark'`?  or was that just an artifact of sticking the `'cdl_none'` check into an already bifurcated conditional (at the time, definitely unnecessarily bifurcated, since the conditionals were right next to each other).

maybe @peetucket can weigh in, since he added the `'cdl_none'` check.  it's been a while since i've worked in dor-rights-auth, so i'm a little fuzzy on it's complex wrangling of the `rightsMetadata` datastream.

if it turns out the bifurcation is necessary (i suspect it isn't, but 🤷), we should probably add a test that this PR would break.

**Review by jmartin-sul — CHANGES_REQUESTED** [review]
i'm not actually sure the code change is problematic, just hitting "request changes" to block merging till my question is answered, so as to prevent inadvertent CDL breakage (and actually even if this did break things, i think it'd just be the faceting in argo for that particular rights combo?).

maybe being overly careful, but feels best to play it safe for rights/CDL stuff.

File: lib/dor/rights_auth.rb
Comment (jmartin-sul) [reply lib/dor/rights_auth.rb:428]: the commit that added the `cdl_none` check:  https://github.com/sul-dlss/dor-rights-auth/commit/342327c74a0a16835dace8c619167968f8892c8e#diff-0a1ca0c5e439f17160515030dfbb57f9209917dd356320f48127bb5345dccc50R421-R422

and its PR:  https://github.com/sul-dlss/dor-rights-auth/pull/45/files?file-filters%5B%5D=.rb#diff-0a1ca0c5e439f17160515030dfbb57f9209917dd356320f48127bb5345dccc50R422-R423

Comment (mjgiarlo) [reply lib/dor/rights_auth.rb:428]: Good call, @jmartin-sul. I'll defer to @peetucket and I'm happy to make changes if needed.

Comment (peetucket) [reply lib/dor/rights_auth.rb:428]: I think this is fine.  I think in the real world, `dark` and `cdl_none` are going to be mutually exclusive, so order of precedence in this method shouldn't matter.

Comment (jmartin-sul) [reply lib/dor/rights_auth.rb:428]: thanks @peetucket!

**Review by jmartin-sul — APPROVED** [review]

**Pull request merged**
